netdeploy/networkTemplates_test.go

```diff
@@ -98,6 +100,111 @@ func TestValidate(t *testing.T) {
 	a.NoError(err)
 }
 
+func TestDevModeValidate(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	// same genesis configuration for all tests.
+	devmodeGenesis := gen.GenesisData{
+		DevMode: true,
+		Wallets: []gen.WalletData{
+			{
+				Stake: 100,
+			},
+		},
+	}
+
+	t.Run("DevMode two relays", func(t *testing.T) {
+		t.Parallel()
+		tmpl := NetworkTemplate{
+			Genesis: devmodeGenesis,
+			Nodes: []remote.NodeConfigGoal{
+				{
+					IsRelay: true,
+				},
+				{
+					IsRelay: true,
+				},
+			},
+		}
+		require.ErrorContains(t, tmpl.Validate(), "devmode configurations may have at most one relay")
```

Comment:
it looks like multiple follower node is allowed in devmode. should we allow that? and maybe add a test case for it?

Reply:
I'll add a test. Not sure if it's useful but it would work.